internal/multistorage/cache/file.go

```diff
@@ -13,15 +13,13 @@ import (
 )
 
 // StatusFile is stored on a disk and therefore shared between all WAL-G processes and commands.
-var StatusFile string
-
-func init() {
+var StatusFile = func() (string, error) {
 	homeDir, err := os.UserHomeDir()
```

Comment:
Let's just move this code [here](https://github.com/wal-g/wal-g/blob/cc850987d1e46cdddb088577744b448e82bb6f5d/internal/multistorage/cache/status_cache.go#L31)?